Source/Scene/LabelCollection.js

```diff
@@ -79,13 +83,52 @@ define([
         });
     }
 
+    function getCSSValue(element, property) {
+        return document.defaultView.getComputedStyle(element, null).getPropertyValue(property);
+    }
+
+    var fontInfoCache = {};
```

Comment:
In the Labels SDF sandcastle the cache grows to 48 elements because of the label that's constantly changing its font size. Add a few more font families and this could get bigger. Maybe reset the cache after it has reached a certain size? Or have it clear based on a timer like `_shaderFrameCount` in `Scene`.

Reply:
I modeled this after the glyphTextureCache on the LabelCollection which appears to grow without shrinking.  This is essentially a "parsed" version of the CSS font string getting the individual properties of the font.  If we don't like the static cache then perhaps it would be better to store family, size, style and weight as individual properties on the Label itself and then just dirty those properties if the font string changes.  You'll take a hit recomputing it for each label even if it shares the same font but that's not a huge deal I don't think.  The fontInfo is used in the rebindAllGlyphs function which can be called pretty often if the text or color of the font is changing so I wanted to avoid reparsing the font each time that function was called.

What do you think?  I'd be find with moving the CSS parsing into Label itself if that would clear things up.

Reply:
That approach of moving everything to Label sounds fine. `label.font` shouldn't really get changed that often anyways.

Reply:
I've addressed this in 07b7d89767 by moving everything over to Label and storing font properties on the Label itself.  The getFontInfo function is now called parseFont and sets the properties on the Label directly instead of returning a strcture.